.gitmodules

```diff
@@ -0,0 +1,15 @@
+[submodule "src/mpid/ch4/netmod/ucx/ucx"]
+	path = src/mpid/ch4/netmod/ucx/ucx
+	url = git://github.com/openucx/ucx
+[submodule "confdb"]
+	path = confdb
+	url = git@github.com:pmodels/confdb
```

Comment:
Do we really want to use SSH-based access to the submodule repos? If I am an external contributor who does not have SSH access to these repos, I will have to manually change these to HTTPS in `.gitmodules` and remember to not included those changes to my own topic commits.

Reply:
Fixed.